Sources/PitchSpeller/Data Structures/_Graph.swift

```diff
@@ -156,6 +156,15 @@ struct _Graph<Weight: Weightedness, Pair: SymmetricPair & Directedness & Hashabl
         self.nodes = nodes
         self.adjacents = adjacents
     }
+
+    // Initializer from phased-out Graph type
+    init (_ graph: __Graph<Node>) {
+        nodes = Set(graph.nodes)
+        adjacents = [:]
+        for edge in graph.edges {
+            adjacents[Pair(edge.source, edge.destination)] = (edge.value as! Weight)
```

Comment:
I'm not super convinced by the need to forcibly cast here (`as!`). Though, is this just a temporary adapter to the deprecated `__Graph`?

Reply:
Could this be nested in an extension as:

```Swift
extension _Graph where Weightedness == Double {
init (_ graph: __Graph<Node>) {
         nodes = Set(graph.nodes)
         adjacents = [:]
         for edge in graph.edges {
             adjacents[Pair(edge.source, edge.destination)] = edge.value
        }
    }
}
```